src/roiextractors/extractors/tiffimagingextractors/scanimagetiffimagingextractor.py

```diff
@@ -235,6 +230,7 @@ def _create_frame_to_ifd_table(
         num_acquisition_cycles: int,
         ifds_per_file: list[int],
         num_frames_per_slice: int = 1,
+        flyback_frames: int = 0,
```

Comment:
Nit: I think it should be called `num_flyback_frames` (more explicit and matches the other args).

Reply:
I agree with this.

Reply:
Changed.